hackathon/views.py

```diff
@@ -48,37 +50,62 @@ def judging(request, hack_id, team_id):
 
     # verify if hackathon is ready to be judged (judging_status == 'open')
     if hackathon.judging_status != 'open':
-        messages.error(request, f"Judging is not open! {hackathon.judging_status}!")
+        messages.error(
+            request, f"Judging is not open! {hackathon.judging_status}!")
         return render(request, 'home/index.html')
 
     # verify that the selected team belongs to the selected hackathon
     if team.hackathon != hackathon:
-        messages.error(request, f"Nice try! That team is not part of the event...")
+        messages.error(
+            request, f"Nice try! That team is not part of the event...")
         return render(request, 'home/index.html')
 
     # check if the judge has already scored the requested team's Project
     project = get_object_or_404(HackTeam, pk=team_id).project
     if not project:
-        messages.error(request, f"The team doesn't have a project yet, check back later...")
+        messages.error(
+            request, f"The team doesn't have a project yet, check back later...")
         return render(request, 'home/index.html')
     judge_has_scored_this = False
     if HackProjectScore.objects.filter(judge=request.user, project=project):
-        messages.error(request, f"Oooops, sorry! Something went wrong, you have already scored that team...")
+        messages.error(
+            request, f"Oooops, sorry! Something went wrong, you have already scored that team...")
         return render(request, 'home/index.html')
-    
+
     if request.method == 'POST':
         # judge score submitted for a team
         for score_category in score_categories:
             score_cat_id = f"score_{score_category.id}"
             team_score = HackProjectScore(
-                created_by = request.user,
-                judge = request.user,
-                project = get_object_or_404(HackTeam, pk=team_id).project,
-                score = request.POST.get(score_cat_id),
-                hack_project_score_category = score_category,
+                created_by=request.user,
+                judge=request.user,
+                project=get_object_or_404(HackTeam, pk=team_id).project,
+                score=request.POST.get(score_cat_id),
+                hack_project_score_category=score_category,
             )
             team_score.save()
-            
+
+        # Number of judges in the Hackathon
```

Comment:
I don't think you need all the  inline comments. Instead I would extract the logic into a separate function to be called here and add the overall logic to the docstring.

Reply:
@stefdworschak done!